### PR TITLE
Credentials nathan register

### PR DESCRIPTION
**Pull request by nthnns**
added login and register from last lab

### CODE REVIEW

**Review by ellaarnold19 — APPROVED** [review]

File: app/src/ui/views/logout.ejs
Comment (feRpicoral) [inline app/src/ui/views/logout.ejs:1]: We don't need a view to logout - just a button on the nav bar

File: app/src/ui/views/login.ejs [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
 </head>
+
 <body>
-<%- include('../partials/header.ejs')%>
-<h1>Test</h1>
+    <%- include('../partials/header.ejs')%>
```
Comment (feRpicoral) [inline app/src/ui/views/login.ejs:11]: This header is something I created when testing out some stuff, it just has "Example Header". We can get rid of it

File: app/src/ui/views/login.ejs [hunk 2]
```diff
@@ -1,12 +1,30 @@
 <!doctype html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
 </head>
+
 <body>
```
Comment (feRpicoral) [inline app/src/ui/views/login.ejs:10]: We need to move to a partial this part, including the bootstrap import and the body open tag

File: app/src/ui/views/login.ejs [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+                    <input type="submit" value="Login" />
+                </form>
+            </div>
+            <p>New User? <a href="/signup">Sign up</a> here</p>
+
+        </main>
 </body>
-</html>
+
+</html>
```
Comment (feRpicoral) [inline app/src/ui/views/login.ejs:30]: We need to move to a partial (footer) the body & html closing tags.

File: app/src/ui/views/register.ejs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<%- include ('../partials/head') %>
```
Comment (feRpicoral) [inline app/src/ui/views/register.ejs:1]: This file doesn't exist

**Review by feRpicoral — CHANGES_REQUESTED** [review]
Left a few comments, but nice work! Other than the specific comments, some overall things:

- Aren't you missing the proper bootstrap classes & the actual bootstrap import?
- Please add some screenshots to the PR here on GH of how it looks
- It's less code to maintain and less duplication if instead of having two different pages for login and signup, we have only one and send the data we need with EJS:
```js
// on endpoint for /login
res.render('views/auth', {method: 'login'}); // add any extra needed data here

// on endpoint for /signup
res.render('views/auth', {method: 'signup'});
```

For this last item I have an example where I did exactly that for one of the labs if you want to take a look.

**Pull request closed without merging**